holoviews/ipython/__init__.py

```diff
@@ -83,6 +83,11 @@ class notebook_extension(extension):
 
     logo = param.Boolean(default=True, doc="Toggles display of HoloViews logo")
 
+    comms = param.ObjectSelector(
+        default='default', objects=['default', 'ipywidgets', 'vscode'], doc="""
+        Whether to render output in Jupyter with the default Jupyter
+        extension or use the jupyter_bokeh ipywidget model.""")
```

Comment:
Should the docstring mention the vscode option?

Reply:
Not sure I want to merge this yet, but it we want to add this hacky workaround then yes. Otherwise we can just wait until VSCode adds ipywidgets 8.0 support and remove the VSCode option.

Reply:
Comment from JL: Don't need this parameter